wallet-new/src/Cardano/Wallet/API/WIP.hs

```diff
@@ -12,10 +12,6 @@ import           Cardano.Wallet.API.V1.Types
 type API = Tags '["WIP"] :>
     (
          "external-wallets"
-                   :> Capture "rootPK" PublicKeyAsBase58
-                   :> Summary "Check if this external wallet is presented in the node."
-                   :> PostCreated '[ValidJSON] (WalletResponse WalletAndTxHistory)
-    :<|> "external-wallets"
                    :> Summary "Creates a new or restores an existing external wallet (mobile client or hardware wallet)."
                    :> ReqBody '[ValidJSON] (New ExternalWallet)
                    :> PostCreated '[ValidJSON] (WalletResponse Wallet)
```

Comment:
Shouldn't that response type be `WalletResponse ExternalWallet` or am I missing something here :thinking: ?

Reply:
No, it should return `Wallet`, for unification. We _can_ think of external wallet as of regular one, but without spending password and with root PK only. :-)